sqlite-std.go

```diff
@@ -20,6 +20,97 @@ type DB struct {
 	tmpFile string
 }
 
+func (db *DB) Collection() (*Collection, error) {
```

Comment:
I would suggest putting these new `*DB` methods in a new file, `sqlite.go`, with no build constraints. By putting them here, they are not available for GopherJS builds (thus the Travis-CI failure).

Reply:
Ah, ok, thanks. I'll update my pull request later.

Reply:
Update, I found the situation again.

    Deck 1446489859874 references non-existent config 0

Both the AnkiDroid and desktop app work without problems with this collection, so this isn't an error. Maybe just write a warning in stderr instead of returning an error here?

Reply:
I guess that means that a default configuration is used in such a case?

Warning to sterr seems reasonable to me.